### PR TITLE
Remove translation keys for removed messages

### PR DESCRIPTION
**Pull request by hoschi**
**What**:
Only found messages are kept in translation files.

**Why**:
When messages get removed in code they should also be removed from translation files.

**How**:
Keep only keys from existing translation objects which are in scanned object.

**Checklist**:

* [ ] Documentation N/A
* [X] Tests
* [X] Ready to be merged 
* [x] Added myself to contributors table

### CODE REVIEW

**Comment by akameco** [comment]
@hoschi Thanks PR :tada:

Try:

```
$ yarn run add-contributor hoschi
```

File: test/json/snapshots/test.js.md [hunk 1]
```diff
@@ -81,9 +117,6 @@ Generated by [AVA](https://ava.li).
       'a.world',
       'b.hello',
       'b.world',
-      'c.hello',
-      'y.hello',
-      'z.hello',
```
Comment (akameco) [inline test/json/snapshots/test.js.md]: Why diff?

Comment (hoschi) [reply test/json/snapshots/test.js.md]: damn, forgot to fix that, sorry. I'll look into it.

**Comment by hoschi** [comment]
@akameco fixed the sort test by adding better fixtures. Also added myself as contributor now worked.

**Comment by akameco** [comment]
@hoschi Great Job👍 Thanks.

**Pull request merged**